_docs_integrate/examples/docker-compose-with-mongodb.yml

```diff
@@ -1,4 +1,4 @@
-version: "3.7"
+# do not use this compose file in production scenarios
```

Comment:
# ATTENTION: Do NOT use this file in public or production scenarios! It may contain insecure or unstable configuration.